cms/navigation/tests/test_navigation_tags.py

```diff
@@ -81,3 +84,52 @@ def test_main_menu_columns_empty_menu(self):
         """Test that main_menu_columns returns an empty list for a None menu."""
         columns = main_menu_columns({}, None)
         self.assertEqual(columns, [])
+
+
+class FooterMenuTemplateTagTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.mock_request = wagtail.coreutils.get_dummy_request()
+        cls.footer_menu = FooterMenuFactory()
+
+        links = [LinkBlockFactory()] * 5
+
+        column_value = LinksColumnFactory(links=links)
+
+        columns = [
+            {
+                "type": "column",
+                "value": column_value,
+            }
+        ] * 3
+
+        cls.footer_menu.columns = columns
+        cls.footer_menu.save()
```

Comment:
this could probably be simplified as

```suggestion
        links = LinkBlockFactory.create_batch(5)
        columns = [
            {
                "type": "column",
                "value": LinksColumnFactory(links=links),
            }
        ] * 3
        
        cls.footer_menu = FooterMenuFactory(columns=columns)
```

https://factoryboy.readthedocs.io/en/stable/reference.html#factory.Factory.create_batch

Reply:
✅ [1067893](https://github.com/ONSdigital/dis-wagtail/pull/111/commits/1067893aa8ff7870288467daf7a4a8d5b3ba8869)